main.go

```diff
@@ -65,6 +70,34 @@ func main() {
 		}, "Could not create WIT client")
 	}
 
+	authClient, err := auth.NewCachedClient(config.GetAuthURL())
+	if err != nil {
+		log.Panic(nil, map[string]interface{}{
+			"url": config.GetAuthURL(),
+			"err": err,
+		}, "could not create Auth client")
+	}
+
+	// Calling the Auth service to generate a fabric8 service account token.
+	// This is needed to call /api/users/ID and 'see' the
+	// email address even if the user has made it 'private'
+	saService := token.NewFabric8ServiceAccountTokenClient(authClient, config.GetServiceAccountID(), config.GetServiceAccountSecret())
+	saToken, err := saService.Get(context.Background())
+	if err != nil {
+		log.Panic(nil, map[string]interface{}{
+			"err": err,
+		}, "could not generate service account token")
```

Comment:
We should really fail here. Not just log and proceed.

Reply:
umm... but log.Panic(...) does exit the program, right?

Reply:
Oh, right!